python/raft-ann-bench/src/raft-ann-bench/plot/__main__.py

```diff
@@ -213,18 +228,15 @@ def inv_fun(x):
 def create_plot_build(
     build_results, search_results, linestyles, fn_out, dataset
 ):
+    bt_80 = [0] * len(linestyles)
```

Comment:
Can we please report 80-89, 90-94, 95-98, 99+? It's really the windows everyone cares about (and the labels should be updated to reflect this).